code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm

```diff
@@ -134,11 +134,8 @@
 			to_chat(user, "<span class='warning'>Something's interfering with [src]'s effects. It's no use.</span>")
 			return
 		A.buffed++
-		A.maxHealth *= 1.5
-		A.health = min(A.maxHealth,A.health*1.5)
-		A.melee_damage_lower = max((A.melee_damage_lower * 2), 10)
-		A.melee_damage_upper = max((A.melee_damage_upper * 2), 10)
+		A.maxHealth *= 1.2
```

Comment:
> no hp increase

increasing HP

Reply:
There's no need to add HP. It just gives more reason for people like Adam and Jcat to scream about it being removed because they don't like that anyone can have fun doing anything other than killing people with toolboxes and mining loot.